src/ildasm/dis.cpp

```diff
@@ -1646,7 +1646,7 @@ BOOL Disassemble(IMDInternalImport *pImport, BYTE *ILHeader, void *GUICookie, md
 
             case ShortInlineBrTarget:
             {
-                char offset = (char) pCode[PC];
+                signed char offset = (signed char) pCode[PC];
```

Comment:
LGTM as long as it works. BTW, char is already signed and I wonder why we need signed keyword explicitly. I understand pCode is declared as BYTE* which is unsigned one, but casting to (char) should work. I guess this might be a bug (or different semantic) in arm compiler that you're using. I couldn't make a simple c++ cutdown to repro this with x64 clang though.